docs/knowledge_base/threat_feed.md

```diff
@@ -13,22 +13,17 @@ The Phylum threat feed provides a curated view into malware being released acros
 
 ## Quickstart
 
-1. Obtain an API key:
+1. Obtain an API from [the Phylum UI](https://app.phylum.io/tokens/list) and set it as follows:
```

Comment:
```suggestion
1. Obtain an [API key](https://docs.phylum.io/docs/api-keys) and set it as follows:
```

While this may be a little less direct, it might be more robust as CLI methods are added and updated on that page.